cmd/agent/common/import.go

```diff
@@ -144,6 +144,19 @@ func ImportConfig(oldConfigDir string, newConfigDir string, force bool) error {
 			continue
 		}
 
+		// Transform if needed AD configuration
+		input, err := ioutil.ReadFile(dst)
```

Comment:
Why not read from `src` and remove the invocation of `copyFile`?

Reply:
@xvello `copyFile` is doing a bunch of additional things than just copying the file, I agree it's not optimal to open the file twice but it's tradeoff with code simplicity

Reply:
Okay. Just need to fix the tests then